Content.Shared/GameObjects/Components/SharedStackComponent.cs

```diff
@@ -137,6 +137,12 @@ public enum StackType
         FloorTileWhite,
         FloorTileDark,
         FloorTileWood,
-        MetalRod
+        MetalRod,
+        PaperRolling,
+        CigaretteFilter,
+        GroundTobacco,
+        GroundCannabis,
+        LeavesTobaccoDried,
+        LeavesCannabisDried
```

Comment:
These can now be removed imo

Reply:
Nah, I still want these to be stacks so they have to be there.